action-api/src/main/java/com/powsybl/action/LoadAction.java

```diff
@@ -44,4 +42,14 @@ public String getLoadId() {
     public String getType() {
         return NAME;
     }
+
+    @Override
+    public LoadActionBuilder convertToBuilder() {
+        return new LoadActionBuilder()
+            .withId(id)
+            .withLoadId(loadId)
+            .withRelativeValue(relativeValue)
+            .withActivePowerValue(activePowerValue)
+            .withReactivePowerValue(reactivePowerValue);
```

Comment:
You can use the get methods instead of using the variables directly, for example:
```suggestion
        LoadActionBuilder builder = new LoadActionBuilder()
            .withId(this.getId())
            .withLoadId(this.getLoadId())
            .withRelativeValue(this.isRelativeValue());
        if (this.getActivePowerValue().isPresent()) {
            builder.withActivePowerValue(this.getActivePowerValue().getAsDouble());
        }
        if (this.getReactivePowerValue().isPresent()) {
            builder.withReactivePowerValue(this.getReactivePowerValue().getAsDouble());
        }
        return builder;
```